distribution/changelog.txt

```diff
@@ -36,6 +36,7 @@
 - Improved: [#16408] Improve --version cli option to report more compatibility information.
 - Improved: [#16740] Allow staff patrol areas to be defined with individual tiles rather than groups of 4x4.
 - Improved: [#16764] [Plugin] Add hook 'map.save', called before the map is about is saved.
+- Change: [#7983] Change when certain G-forces show up in red.
```

Comment:
I think it's better explained here that they show up in red now when they negatively affect the ride's stats.
```suggestion
- Change: [#7983] The red color in the ride graphs now properly corresponds to when they negatively affect a ride’s stats.
```

Reply:
That's not entirely true. As stated, this PR does not change when positive G's are displayed in red, since there is no baseline to compare it to - positive G's never stop giving excitement and don't result in a penalty.

Perhaps we could simply remove the red color for positive G's to prevent any possible confusion?

Reply:
> Perhaps we could simply remove the red color for positive G's to prevent any possible confusion?

Definitely, that's also what was suggested in #7983.

Reply:
I'll do that too. This also means one of the strings defined in `StringIds.h` will be removed (`STR_MAX_POSITIVE_VERTICAL_G_RED = 1349` to be precise`). Should I also remove all translations of said string as part of clean-up duty? :)